addOns/pscanrulesAlpha/src/main/java/org/zaproxy/zap/extension/pscanrulesAlpha/CacheableScanRule.java

```diff
@@ -160,13 +154,11 @@ public void scanHttpResponseReceive(HttpMessage msg, int id, Source source) {
                     // strip off any trailing comma
                     if (directiveToken.endsWith(","))
                         directiveToken = directiveToken.substring(0, directiveToken.length() - 1);
-                    if (logger.isTraceEnabled())
-                        logger.trace("Looking for 'no-store' in [" + directiveToken + "]");
+                    logger.trace("Looking for 'no-store' in [{}]", directiveToken);
                     if (directiveToken.toLowerCase().equals("no-store")) {
-                        if (logger.isDebugEnabled())
-                            logger.debug(
-                                    msg.getRequestHeader().getURI().getURI()
-                                            + " is not storable due to the use of HTTP caching directive 'no-store' in the request or response");
+                        logger.debug(
+                                "{} is not storable due to the use of HTTP caching directive 'no-store' in the request or response",
+                                msg.getRequestHeader().getURI().getURI());
```

Comment:
```suggestion
                                msg.getRequestHeader().getURI().toString());
```

Instead of marking up the same thing a ton of times please just change it throughout.